CIS Scripts/2_Security_Audit_Compliance.sh

```diff
@@ -657,9 +657,9 @@ fi
 Audit2_5_5="$($Defaults read "$plistlocation" OrgScore2_5_5)"
 # If organizational score is 1 or true, check status of client
 if [ "$Audit2_5_5" = "1" ]; then
-CP_disableDiagnostic="$(/usr/sbin/system_profiler SPConfigurationProfileDataType | /usr/bin/grep -c 'allowDiagnosticSubmission = 0')"
+CP_disableDiagnostic="$(python -c 'from CoreFoundation import CFPreferencesCopyAppValue; print CFPreferencesCopyAppValue("AutoSubmit", "com.apple.SubmitDiagInfo")')"
```

Comment:
Is this extending python with c? Could it be python3 -c? Also not sure, but seems there is a couple of this, would it be worth passing it as a module? Would dependencies need to be installed for this?